contracts/hackatom/src/contract.rs

```diff
@@ -108,14 +110,14 @@ pub fn init<S: Storage, A: Api, Q: Querier>(
     // This adds some unrelated event attribute for testing purposes
     let mut ctx = Context::new();
     ctx.add_attribute("Let the", "hacking begin");
-    ctx.try_into()
+    Ok(ctx.try_into()?)
```

Comment:
This construct is needed to convert `StdError` to `HackError`, right?

Another tip that can be added to the migration info.

Reply:
Yeahish: what we really do with this construct is conveert `Result<InitResponse, StdError>` to `Result<InitResponse, HackError>`. It it was the error only, a simple `.into()` would do the job.